libs/application/core/src/types/StateMachine.ts

```diff
@@ -19,6 +19,7 @@ export enum DefaultEvents {
   SUBMIT = 'SUBMIT',
   ABORT = 'ABORT',
   EDIT = 'EDIT',
+  PAYMENT = 'PAYMENT',
```

Comment:
Not a big deal, but not sure it should be part of the default state but rather a custom state for your template instead

Reply:
yes, sorry, that's actually leftovers. good catch